jyutping.schema.yaml

```
@@ -61,14 +61,22 @@ speller:
     - derive/eoi$/eoy/           # 容錯
     - derive/eo/oe/              # 教院式
     - derive/oe(ng|k)$/eo$1/     # 容錯
-    - derive/aa$/a/              # 容錯
```

呢一行有冇必要刪去？因爲呢一條同埋`- derive/aa/a/`其實並唔等價，呢一條只不過係允許開音節嘅aa可以省略爲a，但係閉音節嘅aa唔可以縮寫成a，所以唔係懶音規則。我認爲應該保留此條，以及取消加入下面嘅`- derive/aa/a/`
```suggestion
    - derive/aa$/a/              # 容錯
```